templates/index.html

```diff
@@ -11,7 +11,7 @@
     <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
             <div class="navbar-header">
-                <a class="navbar-brand" href="#">Route Documentation</a>
+                <a class="navbar-brand" href="#">Route Documentation{{#@apiVersion}} - v{{@apiVersion}}{{/@apiVersion}}</a>
```

Comment:
Where is that #@ form documented ? handlebars seems to lack any documentation about that.